### PR TITLE
February Newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
* This newsletter features the sandiegopython-organizers@ email list. There's a thread on the organizers list if you think this shouldn't be the case.
* Features the monthly meetup, saturday meetup and the data science workshop

### CODE REVIEW

File: content/posts/2015-02-02_february-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+popular scientific Python distribution Anaconda. The meetup took place at and
+was sponsored by [NanoImaging Services][].
+
+[january-video]: https://www.youtube.com/watch?v=bJP8NoV7rV4
+[January Meetup]: http://www.meetup.com/pythonsd/events/219205663/
+[Continuum Analytics]: http://continuum.io
+[NanoImaging Services]: http://www.nanoimagingservices.com/
+
+
+[SoCal Linux Expo][socal-linux] (socallinuxexpo.com) <br />
```
Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: PyLadies San Diego along with the other SoCal PyLadies groups will be hosting a booth at ....

File: content/posts/2015-02-02_february-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at Pangea cafe. The goal of the Saturday study group is to learn Python
+together in a collaborative environment. Over the next few weeks, the group
+will feature some data mining/analysis presentations by Eric Busboom. If
+you're going to the data science workshop, check it out!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
+
+
+[Monthly Meetup][] (meetup.com) <br />
```
Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: I would love to do a very rough version of my PyCon talk.

Comment (davidfischer) [inline content/posts/2015-02-02_february-newsletter.md]: Were you thinking something on the order of 5-10 minutes? Or something longer?

Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: On 02/02/2015 10:51, David Fischer wrote:
>
> In content/posts/2015-02-02_february-newsletter.md 
> <https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23947481>:
>
> > +
> > +
> > +Upcoming events
> > +---------------
> > +
> > +[Saturday study group][saturday-meetup] (meetup.com) <br />
> > +San Diego PyLadies and SD Python are continuing the Saturday study group
> > +at Pangea cafe. The goal of the Saturday study group is to learn Python
> > +together in a collaborative environment. Over the next few weeks, the group
> > +will feature some data mining/analysis presentations by Eric Busboom. If
> > +you're going to the data science workshop, check it out!
> > +
> > +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
> > +
> > +
> > +[Monthly Meetup][] (meetup.com) <br />
>
> Were you thinking something on the order of 5-10 minutes? Or something 
> longer?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/pythonsd/pythonsd.org/pull/65/files#r23947481>.
>
Was hoping for about 25 minutes... Ideally a dry run for PyCon with 
enough time to fine tune.


-- 
*Carol Willing*
Developer | Willing Consulting
https://willingconsulting.com

Comment (alaindomissy) [inline content/posts/2015-02-02_february-newsletter.md]: Hi Carol,
Is the topic the same as your poster/demo for the Data Science Day on Feb
28th ?
Thanks
--Alain

On Mon, Feb 2, 2015 at 11:19 AM, Carol Willing <notifications@github.com>
wrote:

> In content/posts/2015-02-02_february-newsletter.md
> <https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23949831>:
>
> > +
> > +
> > +Upcoming events
> > +---------------
> > +
> > +[Saturday study group][saturday-meetup] (meetup.com) <br />
> > +San Diego PyLadies and SD Python are continuing the Saturday study group
> > +at Pangea cafe. The goal of the Saturday study group is to learn Python
> > +together in a collaborative environment. Over the next few weeks, the group
> > +will feature some data mining/analysis presentations by Eric Busboom. If
> > +you're going to the data science workshop, check it out!
> > +
> > +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
> > +
> > +
> > +[Monthly Meetup][] (meetup.com) <br />
>
> On 02/02/2015 10:51, David Fischer wrote: In content/posts/
> 2015-02-02_february-newsletter.md <
> https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23947481>: >
> + > + > +Upcoming events > +--------------- > + > +[Saturday study
> group][saturday-meetup] (meetup.com) <br /> > +San Diego PyLadies and SD
> Python are continuing the Saturday study group > +at Pangea cafe. The goal
> of the Saturday study group is to learn Python > +together in a
> collaborative environment. Over the next few weeks, the group > +will
> feature some data mining/analysis presentations by Eric Busboom. If >
> +you're going to the data science workshop, check it out! > + >
> +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/ > +
> > + > +[Monthly Meetup][] (meetup.com) <br /> Were you thinking something
> on the order of 5-10 minutes? Or something longer? — Reply to this email
> directly or view it on GitHub <
> https://github.com/pythonsd/pythonsd.org/pull/65/files#r23947481>.
> Was hoping for about 25 minutes... Ideally a dry run for PyCon with enough
> time to fine tune.
> … <#14b4bbbe0e066865_>
> -- *Carol Willing* Developer | Willing Consulting
> https://willingconsulting.com
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonsd/pythonsd.org/pull/65/files#r23949831>.
>



-- 
--Alain
619-884-5450

Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: Nope the Data Science Day is from PyCon 2014 poster session.

Here's the link to the PyCon 2015 talk: https://us.pycon.org/2015/schedule/presentation/379/

Comment (treyhunner) [inline content/posts/2015-02-02_february-newsletter.md]: I think we should make an exception from our lightning talk-only schedule for Carol's talk.  How often do you get the opportunity to hear a PyCon preview talk? :smile:

@willingc would you rather give your talk preview for the February or March meetup?

Comment (davidfischer) [inline content/posts/2015-02-02_february-newsletter.md]: Sounds good to me. Just let us know whether you're thinking of the monthly meetup for this month or next.

Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: I would love to say this month. Likely March would be closer to the final version.

Comment (davidfischer) [inline content/posts/2015-02-02_february-newsletter.md]: Would you prefer to give a more raw presentation or a more polished presentation? I completely understand both.

Comment (willingc) [inline content/posts/2015-02-02_february-newsletter.md]: On 02/02/2015 14:56, David Fischer wrote:
>
> In content/posts/2015-02-02_february-newsletter.md 
> <https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23967211>:
>
> > +
> > +
> > +Upcoming events
> > +---------------
> > +
> > +[Saturday study group][saturday-meetup] (meetup.com) <br />
> > +San Diego PyLadies and SD Python are continuing the Saturday study group
> > +at Pangea cafe. The goal of the Saturday study group is to learn Python
> > +together in a collaborative environment. Over the next few weeks, the group
> > +will feature some data mining/analysis presentations by Eric Busboom. If
> > +you're going to the data science workshop, check it out!
> > +
> > +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
> > +
> > +
> > +[Monthly Meetup][] (meetup.com) <br />
>
> Would you prefer to give a more raw presentation or a more polished 
> presentation? I completely understand both.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/pythonsd/pythonsd.org/pull/65/files#r23967211>.
>
Let's do March. We already have a bunch going on in Feb :)

-- 
*Carol Willing*
Developer | Willing Consulting
https://willingconsulting.com

**Comment by davidfischer** [comment]
Any other feedback before I send the newsletter?

**Comment by alaindomissy** [comment]
David,

In the 28th Feb announcement:
-  the "R user group" should really be called "Data Science and R user
group"
- "There will be Python and R tracks, data challenges and a hackathon."
should be:
There will be Python and R tracks, data science exhibits, and data
challenges and a hackathon.

Also I should have confirmation today or tomorrow that 2 other meetups will
participate to the Feb 28th Data Science Day:
http://www.meetup.com/San-Diego-Machine-Learning/
http://www.meetup.com/San-Diego-Predictive-Analytics-Meetup/events/219562858/
Would you consider waiting one more day so we can include this ?

Thanks
--Alain




On Mon, Feb 2, 2015 at 7:05 PM, David Fischer <notifications@github.com>
wrote:

> Any other feedback before I send the newsletter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pythonsd/pythonsd.org/pull/65#issuecomment-72584511>.
>



-- 
--Alain
619-884-5450

**Comment by alaindomissy** [comment]
I made this branch:
https://github.com/pythonsd/pythonsd.org/tree/datascienceworkshop-updatesd
It is based on your FebNewsletter branch. Sorry I am not sure what is the
right process is...
If you have to publish to night I would go with these changes.
Thanks
--Alain

On Mon, Feb 2, 2015 at 7:34 PM, Alain Domissy <alaindomissy@gmail.com>
wrote:

> David,
>
> In the 28th Feb announcement:
> -  the "R user group" should really be called "Data Science and R user
> group"
> - "There will be Python and R tracks, data challenges and a hackathon."
> should be:
> There will be Python and R tracks, data science exhibits, and data
> challenges and a hackathon.
>
> Also I should have confirmation today or tomorrow that 2 other meetups
> will participate to the Feb 28th Data Science Day:
> http://www.meetup.com/San-Diego-Machine-Learning/
>
> http://www.meetup.com/San-Diego-Predictive-Analytics-Meetup/events/219562858/
> Would you consider waiting one more day so we can include this ?
>
> Thanks
> --Alain
>
>
>
>
> On Mon, Feb 2, 2015 at 7:05 PM, David Fischer <notifications@github.com>
> wrote:
>
>> Any other feedback before I send the newsletter?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/pythonsd/pythonsd.org/pull/65#issuecomment-72584511>.
>>
>
>
>
> --
> --Alain
> 619-884-5450
>



-- 
--Alain
619-884-5450

**Comment by davidfischer** [comment]
I merged your commits.

**Pull request merged**